Kernel/Bus/USB/UHCI/UHCIDescriptorTypes.h

```diff
@@ -184,21 +202,21 @@ struct alignas(16) TransferDescriptor final {
     }
 
     // FIXME: For the love of God, use AK SMART POINTERS PLEASE!!
-    TransferDescriptor* next_td() { return m_next_td; }
-    TransferDescriptor const* next_td() const { return m_next_td; }
-    void set_next_td(TransferDescriptor* td) { m_next_td = td; }
+    TransferDescriptor* next_td() { return m_bookkeeping->next_td; }
+    TransferDescriptor const* next_td() const { return m_bookkeeping->next_td; }
+    void set_next_td(TransferDescriptor* td) { m_bookkeeping->next_td = td; }
 
-    TransferDescriptor* prev_td() { return m_prev_td; }
-    TransferDescriptor const* prev_td() const { return m_prev_td; }
-    void set_previous_td(TransferDescriptor* td) { m_prev_td = td; }
+    TransferDescriptor* prev_td() { return m_bookkeeping->prev_td; }
+    TransferDescriptor const* prev_td() const { return m_bookkeeping->prev_td; }
+    void set_previous_td(TransferDescriptor* td) { m_bookkeeping->prev_td = td; }
 
     void insert_next_transfer_descriptor(TransferDescriptor* td)
     {
         m_link_ptr = td->paddr();
         td->set_previous_td(this);
         set_next_td(td);
 
-        // Let's set some bits for the link ptr
+        // Let's set some m_bookkeeping for the link ptr
```

Comment:
The find+replace for bits --> m_bookeeping found at least two comments I've seen

Reply:
I'll get that fixed up, I think I fat fingered the Find+Replace lol